site/examples/ExampleHeader.js

```diff
@@ -9,6 +9,13 @@ var Constants = require('../Constants');
 
 var ExampleHeader = React.createClass({
   render() {
+    const contextDisclamer = this.props.page.fileName === 'ContextExample.js' ? (
```

Comment:
This is less than ideal, but I didn't see a good way to add this without a larger refactoring.